go.mod

```diff
@@ -26,7 +27,7 @@ require (
 )
 
 require (
-	github.com/argoproj/argo-cd/v2 v2.8.3 // indirect
+	github.com/argoproj/pkg v0.13.7-0.20230626144333-d56162821bd1 // indirect
```

Comment:
This library might not be required.  I see`k8s.io/apimachinery/pkg/labels` is what is being imported.

Reply:
I tried removing this library but it gets updated after `go mod tidy`

Reply:
ok. This is marked as indirect dependency.